desktop-widgets/divelistview.cpp

```diff
@@ -922,11 +922,14 @@ void DiveListView::shiftTimes()
 
 void DiveListView::loadImages()
 {
-	QStringList filters = imageExtensionFilters();
+	QStringList m_filters = mediaExtensionFilters();
+	QStringList i_filters = imageExtensionFilters();
+	QStringList v_filters = videoExtensionFilters();
 	QStringList fileNames = QFileDialog::getOpenFileNames(this,
-							      tr("Open image files"),
+							      tr("Open media files"),
 							      lastUsedImageDir(),
-							      tr("Image files (%1)").arg(filters.join(" ")));
+							      tr("Media files (%1);;Image files (%2);;Video files (%3)")
+							      .arg(m_filters.join(" ")).arg(i_filters.join(" ")).arg(v_filters.join(" ")));
```

Comment:
If all arguments are strings, I think the more efficient (and safer) method is:
```
x.arg(string1, string2, string3)
```
The reason is: suppose string1 contains "%1" or something, then
```
x.arg(string1).arg(string2).arg(string3)
```
will do funky things.